.travis.yml

```diff
@@ -13,19 +13,16 @@ matrix:
     - rvm: jruby-1.7
       jdk: openjdk7
     - rvm: jruby-1.7
-      jdk: oraclejdk7
```

Comment:
Why leave behind the `jruby-1.7` line without a `jdk` definition? We're already testing JRuby 1.7 with openjdk7 and oraclejdk8

Reply:
Sorry, missed that. I won't push the change now though as it will just fail again. Let's see what @lpradovera finds out.